src/HomePage/AboutMe/AboutMe.scss

```diff
@@ -0,0 +1,26 @@
+.about-section {
+  display: flex;
+  flex-wrap: wrap;
+  align-content: center;
+  min-height: 100vh;
+  background-color: rgb(158, 158, 168);
+
+  &__header {
+    font-family: "Questrial";
+    font-size: 40px;
+    width: 100%;
```

Comment:
Is this needed? Block elements will take up the whole width of the parent anyway